addon/components/uni-multi-selector.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import layout from '../templates/components/uni-multi-selector';
+
+const { Component, isEmpty } = Ember;
+
+export default Component.extend({
+  layout,
+
+  options: [],
+  name: null,
+  groupValue: null,
+
+  init() {
+    this._super(...arguments);
+
+    if (isEmpty(this.get('groupValue'))) {
+      this.set('groupValue', this.get('options')[0].value);
+    }
+  },
+
+  actions: {
+    hasChanged(value) {
+      this.set('groupValue', value);
```

Comment:
I would also suggest having an optional custom function that is executed when the value changes. It might be useful in the future.

```js
actions: {
     hasChanged(value) {
         this.set('groupValue', value);
          
         this.get('onChange')();
     }
}

```